hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -125,21 +150,6 @@
       milestone: V2
       tests: ["spi_device_extreme_fifo_size"]
     }
-    {
-      name: mode
-      desc: '''TODO :only support fw mode now'''
-      milestone: V2
-      tests: []
-    }
-    {
```

Comment:
@eunchan I don't think we have ECC in the memory, so I remove this. Please confirm.

Reply:
THanks